shop-the-look/README.md

```diff
@@ -0,0 +1,404 @@
+# 👘  Shop The Look 
+
+## A multimodal search engine for finding your new favorite outfit
+
```

Comment:
As we discussed, would be great to see a system diagram up at the top for folks trying to quickly grok how the app is arrayed / architecture / clouds involved, etc.